helper/TestdataHelper.js

```diff
@@ -1,20 +1,74 @@
 const prodConfig = require('../testSettings/prod.json')
 const countryCode = require("../testSettings/countryCode.json")
 
+let primaryLanguageFooters = require("../footers/primaryLanguageFooters.json");
+let secondLanguageFooter = require("../footers/secondLanguageFooter.json");
+
 exports.TestDataHelper = {
 
-    setTestCountry: function(value){
-        
+    setTestCountry: function (value) {
+
         let countryUrl = prodConfig["url"][value];
-    
+
         if (countryUrl === undefined) throw `Country ${value} is not found`;
-        
+
         let countryUnderTest = {
             name: value,
             code: countryCode[value.toUpperCase()],
             url: countryUrl,
-          };
-          globalThis.countryUnderTest = countryUnderTest;
+        };
+        globalThis.countryUnderTest = countryUnderTest;
+    },
+
+     getPrimaryLanguageFooterLinks: function (countryCode) {
+        let marketFooters = primaryLanguageFooters[countryCode];
+        if (marketFooters === undefined)
+            throw `The country code ${countryCode} does not exist`;
+        return marketFooters;
     },
 
+     getSecondLanguageFooterLinks: function (countryCode) {
+        let marketFooters = secondLanguageFooter[countryCode];
+        if (marketFooters === undefined)
+            throw `The country code ${countryCode} does not exist`;
+        return marketFooters;
+    },
+
```

Comment:
Can we use this opportunity to merge this into one function? we can do switch depending on the context but have a primary as default if nothing is specified

Reply:
Good call. I have done this